src/index.css

```diff
@@ -1,4 +1,338 @@
+@import url("https://fonts.googleapis.com/css?family=Montserrat");
+@import url("https://fonts.googleapis.com/css?family=Roboto Slab'");
+
 body {
   margin: 0;
   padding: 0;
+  background-color: #e7e3e3;
+}
+
+ul {
+  list-style-type: none;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
+
+.bookStore {
+  width: 90vw;
+  height: 90vh;
+  margin: 3vh 5vw;
+  border-radius: 4px;
+  box-shadow: 0 0 60px 0 rgba(0, 0, 0, 0.05);
+  background-color: whitesmoke;
+}
+
+.active {
+  color: black;
+  font-weight: 700;
+}
+
+.navBar {
+  display: flex;
+  flex-direction: row;
+  padding-left: 5rem;
+  padding-bottom: 1.5rem;
+  background-color: whitesmoke;
+}
+
+.links {
+  display: flex;
+  justify-content: space-between;
+  padding-top: 1rem;
+  width: 15vw;
+}
+
+.book-container {
+  display: flex;
+  flex-direction: row;
+  width: 60vw;
+  margin: 1.313rem 3.25rem 0;
+  padding: 0.8rem 9.188rem 1.625rem 1.688rem;
+  padding-bottom: 2rem;
+  border-radius: 4px;
+  border: solid 1px var(--white-two);
+  background-color: white;
+  margin-left: 10%;
+}
+
+.inputForm {
+  width: 72vw;
+  height: 4.813rem;
+  margin: 1.313rem 4.85rem 0;
+  padding: 0;
+  padding-bottom: 3rem;
+  border-top: solid 1px #b4b0b0;
+  background-color: #e7e3e3;
+  margin-left: 13%;
+}
```

Comment:
- [ ] Why don't you align the `form` properly and make the form to be horizontal on the page so as to match the design guideline and also as suggested by the previous reviewer?😄

Your Implementation | Zeplin Design
:---:|:---:
![](https://i.imgur.com/e2Sn3be.png) | ![](https://i.imgur.com/jvF0e9O.png)

Reply:
The form is well aligned on my own side.